apps/web_delivery/mix.exs

```diff
@@ -36,4 +31,11 @@ defmodule WebDelivery.Mixfile do
       {:rollbax, "~> 0.8.0"},
     ]
   end
+
+  defp elixirc_paths(:test), do: ["lib", "web", "test/support"]
+  defp elixirc_paths(_),     do: ["lib", "web"]
+
+  def application do
+    [mod: {WebDelivery.Application, []}, extra_applications: [:logger]]
+  end
```

Comment:
Many languages (e.g., Ruby) force you to group your public and private methods, or else go through some ceremony (e.g., `private :this_method, :that_method`) that doesn’t help in the readability department. It’s a holdover from the days when source files were printed and cut in half to document the class API.